PSOperations.podspec

```diff
@@ -30,10 +30,11 @@ Pod::Spec.new do |s|
 	s.homepage	= "https://github.com/pluralsight/PSOperations"
 	s.license	= { :type => 'MIT' }
 	s.author	= "Matt McMurry", "Mark Schultz"
-	s.platform     	= :ios, '8.0'
+    s.ios.deployment_target = '8.0'
+    s.tvos.deployment_target = '9.0'
```

Comment:
Could you add
`s.watchos.deployment_target = '2.0'`
and
`s.osx.deployment_target = "10.11"`?